pb_plugins/protoc_gen_dcsdk/enum.py

```diff
@@ -35,11 +42,19 @@ def collect_enums(
             package,
             enums,
             template_env,
+            docs,
             parent_struct=None):
         _enums = {}
 
+        enum_id = 0
         for enum in enums:
-            _enums[enum.name] = Enum(
-                plugin_name, package, template_env, enum, parent_struct)
+            _enums[enum.name] = Enum(plugin_name,
+                                     package,
+                                     template_env,
+                                     enum,
+                                     docs['enums'][enum_id] if docs else None,
+                                     parent_struct)
+
+            enum_id = enum_id + 1
```

Comment:
`enum_id += 1` ? :smile:

Reply:
Oh, I knew `enum_id++` was not Python, but I did not think about that :laughing:

Reply:
Hehe.